prow/plugins/bugzilla/bugzilla.go

```diff
@@ -377,6 +453,20 @@ To reference a bug, add 'Bug XXX:' to the title of this pull request and request
 					response += " The bug has been updated to refer to the pull request using the external bug tracker."
 				}
 			}
+			// if bug is valid and qa command was used, identify qa contact via email
+			if e.command == qaCommand {
+				query := &emailToLoginQuery{}
+				email := bug.QAContactDetail.Email
```

Comment:
What if there is no QA contact or there is no email listed?

Reply:
No QA contact would result in a panic (nil pointer dereference) and no email for the contact would result in no results from the GH query. I'll add some checks for these cases.